app/views/menus/show.html.haml

```diff
@@ -0,0 +1,13 @@
+.row
+  - @menu.menu_items.group_by(&:menu_type).each do |menu_items|
```

Comment:
too much for a view `@menu.menu_items.group_by(&:menu_type)`
also this is a really nice exercise for you too. Please create facade for show action.
there are some examples for ya: https://medium.com/kkempin/facade-design-pattern-in-ruby-on-rails-710aa88326f